src/Kernel32/Kernel32+PLATFORM_ID.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Contains the <see cref="PLATFORM_ID"/> nested type.
+    /// </summary>
+    public static partial class Kernel32
+    {
+        /// <summary>
+        /// The <see cref="PLATFORM_ID"/> structure contains operating system version information.
+        /// </summary>
+        public enum PLATFORM_ID
```

Comment:
Alt suggestions for enum name: `PlatformId`, `OSPlatformId`

Reply:
I think `OSPlatformId` may be the best, as "Platform ID" suggests (to me) CPU architecture. `PLATFORM_ID` being all-caps also suggests the term actually came verbatim from the header files, which it didn't from what I can tell.